.github/workflows/python-test-package.yml

```diff
@@ -4,33 +4,53 @@
 name: Python package
 
 on:
-  workflow_dispatch:
-    branches: [ $default-branch ]
   push:
-    branches: [ $default-branch ]
-  pull_request:
-    branches: [ $default-branch ]
+    branches:
+      - conda-1
```

Comment:
Just to trigger it on my branch (it wasn't running, though it could be due to too many commits at the same time :laughing: I can flail too)